<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartSettingsDialog</class>
 <widget class="QDialog" name="ChartSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/resources/icons/chart_tools.png</normaloff>:/resources/icons/chart_tools.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="accessibleName">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
    position: absolute;
    top: -0.5em;
}

QTabWidget::tab-bar {
    alignment: center;
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid #C4C4C3;
    border-bottom-color: #C2C7CB; /* same as the pane color */
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 8ex;
    padding: 2px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #fafafa, stop: 0.4 #f4f4f4,
                                stop: 0.5 #e7e7e7, stop: 1.0 #fafafa);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}</string>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="energyTab">
        <attribute name="title">
         <string>Energy Chart</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Color - Linea</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="energyXGrid">
             <property name="text">
              <string>X Grid</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="energyYGrid">
             <property name="text">
              <string>Y Grid</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="energyYAxis">
             <property name="text">
              <string>Y Axis</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="energyLineWidth">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="energyXAxis">
             <property name="text">
              <string>X Axis</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>Grosor - Linea</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="energyColor">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
        <zorder>verticalSpacer_3</zorder>
        <zorder>verticalSpacer_4</zorder>
       </widget>
       <widget class="QWidget" name="pitchChart">
        <attribute name="title">
         <string>Pitch Chart</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Color - Linea</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="pitchXAxis">
             <property name="text">
              <string>X Axis</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="pitchYGrid">
             <property name="text">
              <string>Y Grid</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="pitchYAxis">
             <property name="text">
              <string>Y Axis</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="pitchXGrid">
             <property name="text">
              <string>X Grid</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="pitchLineWidth">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Grosor - Linea</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pitchColor">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
        <zorder></zorder>
        <zorder>verticalSpacer_2</zorder>
        <zorder>verticalSpacer</zorder>
       </widget>
       <widget class="QWidget" name="spectrogramTab">
        <property name="accessibleName">
         <string/>
        </property>
        <attribute name="title">
         <string>Spectrogram Chart</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="2" column="0">
            <widget class="QLabel" name="label_37">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="spectrogramColorMap">
             <item>
              <property name="text">
               <string>Spectral</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Jet</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Grey</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hot</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cold</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hues</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="colormapImage">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="spectrogramXAxis">
             <property name="text">
              <string>X Axis</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="spectrogramYAxis">
             <property name="text">
              <string>Y Axis</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Compretion Factor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="valueLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="muLaw">
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="restoreButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Restore</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
